newparp/templates/account/reset_password.html

```diff
@@ -0,0 +1,68 @@
+{% extends "base.html" %}
+{% block scripts: %}<script>
+    var strength = {
```

Comment:
This code might be better placed in the main JS file, but it doesn't matter too much for now.

Reply:
I will look into moving that to the main JS file.